### DISCUSSION TITLE
能否纯用json schema实现动态请求（依赖表单的值）

### DISCUSSION THREAD

**Discussion by cwtuan** [alibaba/formily#1369] [Q&A 2.x] [answered]
比如我有两个Select，分别是SelectA、SelectB
SelectA的选择了某个值后，触发了SelectB的dataSource动态请求（其中请求参数依赖了SelectA的值）

想实现这个效果，能否能使用纯json schema配置出来呢？
看了文档好像都需要写代码的

**Comment by janryWang** [comment, marked as answer]
文档里都有json schema用法的，文档要看仔细，值得说明的是，formily没有内置任何发请求的能力，所以只要涉及异步的逻辑，推荐都是走函数态表达式，具体看看这篇文档 https://v2.formilyjs.org/zh-CN/guide/advanced/business-logic
